### PR TITLE
Add post about ASP.NET Core logging provider for App Service

### PR DESCRIPTION
**Pull request by mderriey**

### CODE REVIEW

**Comment by mderriey** [comment]
See the rendered version over at 👉 <https://github.com/mderriey/mderriey.github.io/blob/add-post-about-aspnet-core-logging-provider-for-app-service/_posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md>

File: _posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+We'll see later how we can take advantage of this to enable it when running the app locally.
+
+The next interesting bit is discovering that the App Service file logger writes log entries to files in the `%HOME%\LogFiles\Application` directory, as seen [in the code](https://github.com/dotnet/extensions/blob/3dc5e9a24865ab84fce6fc078fce4bd7cfcab5c7/src/Logging/Logging.AzureAppServices/src/FileLoggerConfigureOptions.cs#L23).
+This explains why those entries show up in the Log stream feature of App Service, as the official documentation points out that "information written to files ending in .txt, .log, or .htm that are stored in the /LogFiles directory (d:/home/logfiles) is streamed by App Service" ([source](https://docs.microsoft.com/en-us/azure/app-service/troubleshoot-diagnostic-logs#stream-logs)).
```
Comment (staff0rd) [inline _posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md:73]:
```suggestion
This explains why those entries show up in the Log stream feature of App Service, as the [official documentation points out](https://docs.microsoft.com/en-us/azure/app-service/troubleshoot-diagnostic-logs#stream-logs) that:

> ...information written to files ending in .txt, .log, or .htm that are stored in the /LogFiles directory (d:/home/logfiles) is streamed by App Service.
```

File: _posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md [hunk 2]
```diff
@@ -0,0 +1,532 @@
...
+1. This configuration change notification is also picked up by the options system through the configuration-to-options adapter, which triggers a rebuild of the `LoggerFilterOptions` value;
+1. The options system executes all the registered configuration actions for `LoggerFilterOptions`;
+1. One of these configuration actions, as seen earlier, adds a logging filtering rule for the file system logging provider, using the minimum log level option configured in step 1;
+1. The options system notifies change subscribers about the new constructed value;
+1. The logging stack handles that notification and refreshes its filters; and finally
+1. The filter that targets the App Service file system logging provider ensures only entries with a matching log level get written to disk.
+
+Phew.
+We made it 🎉!
+Or did we?
```
Comment (staff0rd) [inline _posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md:356]: Won't markdown render these as a single line? Do you need double line breaks?

Comment (mderriey) [reply _posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md:356]: No, they will be on a single line, which is what I was going for.

See the rendered version at <https://github.com/mderriey/mderriey.github.io/blob/add-post-about-aspnet-core-logging-provider-for-app-service/_posts/2020-08-08-a-look-at-the-aspnet-core-logging-provider-for-app-service.md#how-to-customise-it>

**Pull request merged**